src/benchmark_register.cc

```diff
@@ -461,16 +463,78 @@ int Benchmark::ArgsCnt() const {
   return static_cast<int>(args_.front().size());
 }
 
+void Benchmark::Init() {}
```

Comment:
this should be in benchmark.cc, right?

Reply:
All Benchmakr:: member functions are already in benchmark_register.cc, maybe we shall use a seperate commit to move them to benchmark.cc

Reply:
huh. yeah, probably move benchmark member functions to benchmark.cc and the stuff there to benchmark_main.cc or something.